packages/core/src/components/popover/popover.md

```diff
@@ -1,56 +1,142 @@
+  Omitting the `target` prop will produce an error.
+  ```tsx
+  <Popover content={<Content />} target={<Button text="Open" />} />
+  ```
 
-- `-start`
-- <small>_(nothing)_</small>
-- `-end`
+1. Provide one or two `children`. Omitting a `target` element will produce an error.
+  ```tsx
+  <Popover>
+      <Button text="Open" />
+      <Content />
+  </Popover>
+  ```
 
-For `top` and `bottom`, `-start` means left and `-end` means right. For `left` and `right`, `-start` means top and `-end` means bottom.
+1. It is possible to mix the two: provide the `content` prop and one React child as the target.
+  (Using the `target` prop with `children` is not supported and will produce a warning.)
+  ```tsx
+  <Popover content={<Content />}>
+      <Button text="Open" />
+  </Popover>
+  ```
 
-Therefore, `top-start` places the Popover along the top edge of the target and their left sides will be aligned.
-And `right-end` places the Popover along the right edge with their bottom sides aligned.
+The __target__ acts as the trigger for the popover; user interaction will show the popover based on
+`interactionKind`. The __content__ will be shown in the popover itself. The popover's will always be
+positioned on the page next to the target; the `position` prop determines the relative position (on
+which side of the target).
 
-`auto` will choose the best suitable placement given the Popover's position within its boundary element.
+Internally, the provided target is wrapped in a `span.pt-popover-target`. This in turn is wrapped in a `span.pt-popover-wrapper`:
 
-<div class="pt-callout pt-intent-primary pt-icon-info-sign">
-    Read more in [the Popper.js Placement documentation](https://popper.js.org/popper-documentation.html#Popper.placements).
+```tsx
+<span class="pt-popover-wrapper">
+    <span class="pt-popover-target">
+        <Button text="My target" />
+    </span>
+</span>
+```
+
+The extra `pt-popover-wrapper` is present so that both the popover and target will be wrapped in a single element when rendering popovers [inline](#core/components/popover.inline-popovers):
+
+```tsx
+<span class="pt-popover-wrapper">
+    <span class="pt-popover-target">
+        <Button text="My target" />
+    </span>
+    <!-- Popover HTML goes here -->
+</span>
+```
+
+<div class="pt-callout pt-intent-warning pt-icon-warning-sign">
+    <h5>Button targets</h5>
+    Buttons make great popover targets, but the `disabled` attribute on a `<button>` blocks all
+    events, which interferes with the popover functioning. If you need to disable a button that
+    triggers a popover, you should use [`AnchorButton`](#core/components/button.anchor-button) instead.
+    See the [callout here](#core/components/button.javascript-api) for more details.
 </div>
 
-@## Modifiers
+```tsx
+const { Button, Intent, Popover, PopoverInteractionKind, Position } = "@blueprintjs/core";
+
+export class PopoverExample extends React.Component<{}, {}> {
```

Comment:
with latest React types, you can omit `<{}, {}>` and this will be valid TSX __and valid JSX!__